hera_mc/cm_utils.py

```diff
@@ -135,6 +150,29 @@ def _get_astropytime(_date, _time=0):
     return return_date
 
 
+def get_date_from_pair(d1, d2, ret='earliest'):
+    if d1 is None and d2 is None:
+        return None
+    retdat = None
+    if ret == 'earliest':
+        if d1 is None:
+            retdat = d2
+        elif d2 is None:
+            retdat = d1
+        else:
+            retdat = d1 if d1 < d2 else d2
+    elif ret == 'latest':
+        if d1 is None:
+            retdat = d1
+        elif d2 is None:
+            retdat = d2
+        else:
+            retdat = d1 if d1 > d2 else d2
+    else:
+        warnings.warn("No return comparison value given.")
```

Comment:
Does it make sense for this condition to be a warning? It seems likely to me that it would be better to raise a ValueError if `ret` is not one of the known values.